### DISCUSSION TITLE
Unknown Publisher for Windows 3.1.1 MSI

### DISCUSSION THREAD

**Discussion by ramblin-rose** [apache/couchdb#3622] [Q&A]
Greetings,

First, thanks for the great works that make CouchDB awesome, and secondly apologies if this question is off-topic.

Currently running the Windows 3.1.1 MSI Windows Defender complains about "Unknown Publisher", whereas the 2.3.1 MSI  does not suffer the same fate. In addition, the download link from https://couchdb.apache.org/#download lands at neighborhood.ie, which appears to be a consultant site.  

To me, these are red flags. I am asking, is neighborhood.ie and the publisher issue legitimate, and is it in part related to [#3479](https://github.com/apache/couchdb/issues/3479) and the bintray irritation  [#3478](https://github.com/apache/couchdb/issues/3478)

Thank you

**Comment by wohali** [comment]
The people at `neighbourhood.ie` sponsor the binary packages and provide the downloads. The co-head of that company is in fact the VP of the Apache CouchDB project, Jan Lehnardt @janl .

It is 100% because of #3479 that WIndows does not display a publisher for the package. Nothing to do with #3478 which has been resolved.

**Reply by ramblin-rose** [reply]
Thank you very much for the response and clarification.
